microsoft/testsuites/core/azure_image_standard.py

```diff
@@ -518,6 +518,28 @@ def verify_yum_conf(self, node: Node) -> None:
         else:
             raise SkippedException(f"Unsupported distro type : {type(node.os)}")
 
+    @TestCaseMetadata(
+        description="""
+        Verify if there is any issues in and after 'yum update'
+
+        Steps:
+        1. Run 'yum update -y' command.
+        2. Get the output of dmesg to see if the VM is still in good state.
+        """,
+        priority=2,
+        requirement=simple_requirement(supported_platform_type=[AZURE, READY]),
+    )
+    def verify_yum_update(self, node: Node) -> None:
+        dmesg = node.tools[Dmesg]
+        if isinstance(node.os, Fedora):
+            node.execute(
+                "yum update -y",
+                sudo=True,
+            )
+            dmesg.get_output(force_run=True)
+        else:
+            raise SkippedException(f"Unsupported distro type : {type(node.os)}")
+
```

Comment:
We need reboot to check whether it can be bootable with new kernel

Reply:
Used node reboot for VM's health check.